src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -83,6 +131,24 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
 		content.add(btnRunCommand, c);
 	}
 
+	private void importCommands() {
+		parser = new JsonParser();
```

Comment:
This parser could be a local variable (it is used this way). It would be better if the JsonParser object could be reused (then it can be a field).